specification/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,41 @@
+# Commitment Discount Type
+
+A commitment-based discount is a contractual commitment in exchange for a discounted unit price. These discounts may be available for spend-based commitments or resource-based commitments, depending on the Provider's offerings.
+
+This column determines whether the commitment is based on a minimum spend or minimum usage.
```

Comment:
We decided to remove the word minimum to make this more general / future-proof.

Reply:
Covered in the other comment.